src/solver/conv_asm_implicit_gemm_wrw_gtc_dynamic_xdlops.cpp

```diff
@@ -147,19 +147,19 @@ static inline int GetImplicitGemmWrwGTCDynamicXdlopsGemmkSplits(
 static inline std::tuple<int, int> get_grid_size(const ConvolutionContext& ctx,
                                                  const TunableImplicitGemmGTCDynamic_t* tunable)
 {
-    int k = ctx.n_inputs;
-    int c = ctx.n_outputs;
-    int y = ctx.kernel_size_h;
-    int x = ctx.kernel_size_w;
+    const auto k = ctx.n_inputs;
```

Comment:
This creates a copy of `ctx.inputs` in the stack instead of alias (that costs nothing).